cvat-core/src/enums.ts

```diff
@@ -76,37 +76,36 @@ export enum Source {
 }
 
 export enum LogType {
```

Comment:
I don't see `create:task`, `create:project`, `sort:task`, `filter:task`. Are we going to handle such events on UI?